docs/source/known-issues.rst

```diff
@@ -23,6 +23,49 @@ HIP
 
   gcc 7, 9, and later do not have this issue.
 
+SYCL
+====
+
+- Multiple of the Kokkos algorithm functions use third-party libraries like oneDPL.
```

Comment:
"Multiple of" sounds weird to me.
Maybe "several" or "a number of"?  Maybe a native English speaker can advise here.

Reply:
4e016a2ccc4ecd462a87709762e6009f0327ae91